### PR TITLE
Fixes cyborg screwdriver and wirecutters having no icon

### LINKED ISSUE #16994: Engiborg wirecutters and screwdriver have no sprite
HELP: redacted (?) (PP) (VV) (SM) (FLW) (TP) (REJT): I have no Icon for powered screwdirver or wirecutters, wierd 

Reply PM from-redacted: So I have the engiborg storage open, and it looks normal, exceopt the images for screwdriver and wirecutters are missing, the hoverover still works and they are still usable, but no image, same in hand

### PR DESCRIPTION
**Pull request by ChangelingRain**
really, really shitty overrides
Fixes #16994

### CODE REVIEW

File: code/game/objects/items/nuke_tools.dm [hunk 1]
```diff
@@ -65,12 +65,8 @@
 	desc = "A screwdriver with an ultra thin tip."
 	icon = 'icons/obj/nuke_tools.dmi'
 	icon_state = "screwdriver_nuke"
-	item_state = "screwdriver_nuke"
 	toolspeed = 2
 
-/obj/item/weapon/screwdriver/nuke/New()
```
Comment (ChangelingRain) [inline code/game/objects/items/nuke_tools.dm]: Holy crap this is awful

**Comment by KorPhaeron** [comment]
>kor's fault

**Comment by ChangelingRain** [comment]
Don't merge things that aren't reviewed reee

File: code/game/objects/items/nuke_tools.dm [hunk 2]
```diff
@@ -65,12 +65,8 @@
...
-//to skip screwdriver icon update
```
Comment (ChangelingRain) [inline code/game/objects/items/nuke_tools.dm]: Holy crap this is awful

**Pull request merged**